### DISCUSSION TITLE
Ender 3 MAX

### DISCUSSION THREAD

**Discussion by BEprints** [mriscoc/Ender3V2S1#538] [Q&A]
Hi I have built a Ender 3 Max with a V2 screen and CR Touch on the original Max print head.

Will changing the build volume and probe offset in the advance setting work or do I need to compile new firmware?

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
https://github.com/mriscoc/Ender3V2S1/wiki/Configuration-files

**Reply by BEprints** [reply]
Thanks for that. I have another issue my bed size is 310mm x 320mm my probe offset is +50x -14y but to get it to center on the bed I need to set y offset to -2

**Comment by Kevin286** [comment]
> Hi I have built a Ender 3 Max with a V2 screen and CR Touch on the original Max print head.
> 
> Will changing the build volume and probe offset in the advance setting work or do I need to compile new firmware?

Hello iam Kevin from Germany. I tired to install the Firmware but even with the Guides it doenst Work.
Is anyone here who gets it working?

**Reply by mriscoc** [reply]
https://github.com/mriscoc/Ender3V2S1/wiki#requirements

**Comment by Kevin286** [comment]
Hy, thanks for fast answer. My Ender 3 Hs a 4.2.2 Mainboard where can i see the SOC? Regards

**Reply by mriscoc** [reply]
What color screen do you have?
SoC is only visible by opening the motherboard compartment and checking the largest chip.

**Comment by Kevin286** [comment]
Okay i will look later what Soc i have. The screen i mean is a DACAI, i will check it later to.

**Comment by Kevin286** [comment]
Sorry for my late answer. I got an DACAI Screen and the SoC is an STM32F103. Regars

**Reply by mriscoc** [reply]
If you have a silent board (TMC2208 drivers), just install the Ender3V2 UBLBLT version firmware and follow the calibration guide to adjust your physical settings https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Comment by Kevin286** [comment]
Hello thanks for quick help. Now Firmware is installed. Leveling also works but not when i start a Print. I got the same firmware on my ender 3v2 and there the Printer starts leveling before the print begins.
In my G code i added also G29. Can you help me? Regards

**Reply by mriscoc** [reply]
G29 P1

**Comment by Kevin286** [comment]
; Ender 3 MAX Custom Start G-code
G92 E0 ; Reset Extruder
M190 S60 ; preheat bed for abl
G28 ; Home all axes
G29 ; Auto bed-level (BL-Touch)
;*** Start Preheating ***
M190 S{material_bed_temperature_layer_0} ; heat to setting 
M109 S{material_print_temperature_layer_0} ﻿T0 ; heat to setting
;*** End Preheating ***
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X0.1 Y20 Z0.3 F5000.0 ; Move to start position
G1 X0.1 Y200.0 Z0.3 F1500.0 E15 ; Draw the first line
G1 X0.4 Y200.0 Z0.3 F5000.0 ; Move to side a little
G1 X0.4 Y20 Z0.3 F1500.0 E30 ; Draw the second line
G92 E0 ; Reset Extruder
G1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed
G1 X5 Y20 Z0.3 F5000.0 ; Move over to prevent blob squish

**Reply by dfinberg** [reply]
G29 is not a valid UBL command, only bilinear. Try G29 A and see the marlin reference pages. https://marlinfw.org/docs/gcode/G029-ubl.html

**Reply by Kevin286** [reply]
Perfect, thank you i will try it later.
